### PR TITLE
fix: walletd update to new config

### PR DESCRIPTION
**Pull request by alexfreska**
- Fixed an issue where the app would crash because it was using the old configuration format. Closes https://github.com/SiaFoundation/walletd/issues/163

### CODE REVIEW

**Comment by alexfreska** [comment]
* **#88** <a href="https://app.graphite.dev/github/pr/SiaFoundation/desktop/88?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alexfreska and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by n8maninger — APPROVED** [review]

File: walletd/renderer/components/useConfigData.tsx [hunk 1]
```diff
@@ -16,12 +16,14 @@ export type Config = {
...
-  consensus: {
-    network: 'mainnet' | 'zen'
-    gatewayAddress: string
+  syncer: {
+    address: string
     bootstrap: boolean
     enableUPnP: boolean
   }
+  consensus: {
+    network: 'mainnet' | 'zen' | 'anagami'
```
Comment (n8maninger) [inline walletd/renderer/components/useConfigData.tsx:25]: Don't enable `anagami` in the UI. It's not a testnet we want most users to use.

**Review by n8maninger — CHANGES_REQUESTED** [review]

File: walletd/renderer/components/useConfigData.tsx (hunk 1 above)
Comment (alexfreska) [reply walletd/renderer/components/useConfigData.tsx:25]: 👍🏼 removed

Comment (alexfreska) [reply walletd/renderer/components/useConfigData.tsx:25]: I left the types, but removed the dropdown option.

**Review by n8maninger — APPROVED** [review]

**Pull request merged**